### DISCUSSION TITLE
Bounded accuracy and loss "plateau"

### DISCUSSION THREAD

**Discussion by gastodler** [open-mmlab/mmocr#1951] [Q&A]
Hi, 

I'm training a custom SAR in Arabic. 

I've tried many directions, but I'm stuck where, and my loss get  into a "plateau", and my accuracy is bounded around 0.6 (word level). 

I'd like to know where my problem is: 
1. In the training set
2. in my dict
3. in my learning rate
4. somewhere else / all together

I've tried to check specific areas. for example, I have an image of ("_&_") symbol in my training set, but when I check it using one of my trained models -> they don't recognize it right. my dict has this symbol in it ("_&_"), but I'm still not sure that it says my dict is "correct". I have some similar issues with numbers and other symbols. 

Any help would be appreciated.
